### PR TITLE
Add charts MVP

### PR DESCRIPTION
**Pull request by erichli1**
Add charts MVP, including 2 static charts (one hard-coded, one from CSV) and 1 dynamic chart with form.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/CharityDataServlet.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                programVote = datastore.get(charityKey);
+            } catch (EntityNotFoundException e) {
+                programVote = new Entity("Program", charity);
+                programVote.setProperty("program", charity);
+                programVote.setProperty("votes", (long) 0);
+            }
+
+            long currentVotes = (long) programVote.getProperty("votes");
+
+            if (currentVotes <= Long.MAX_VALUE - 1) {
```
Comment (hdhabu) [inline portfolio/src/main/java/com/google/sps/servlets/CharityDataServlet.java:87]: any specific reason for this limit check?

File: portfolio/src/main/webapp/script-charts.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    const today = new Date(date.getFullYear(), date.getMonth(), date.getDate());
+
+    const data = new google.visualization.DataTable();
+    data.addColumn('string', 'Order');
+    data.addColumn('string', 'Position')
+    data.addColumn('date', 'Start');
+    data.addColumn('date', 'End');
+        data.addRows([
+            ['1', 'Intern at Boston Housing Authority', new Date(2020, 2, 20), new Date(2020, 4, 18)],
+            ['1', 'Google STEP Intern', new Date(2020, 4, 18), today],
```
Comment (hdhabu) [inline portfolio/src/main/webapp/script-charts.js:19]: '2'?

File: portfolio/src/main/webapp/script-charts.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    const date = new Date();
+    const today = new Date(date.getFullYear(), date.getMonth(), date.getDate());
+
+    const data = new google.visualization.DataTable();
+    data.addColumn('string', 'Order');
+    data.addColumn('string', 'Position')
+    data.addColumn('date', 'Start');
+    data.addColumn('date', 'End');
+        data.addRows([
+            ['1', 'Intern at Boston Housing Authority', new Date(2020, 2, 20), new Date(2020, 4, 18)],
```
Comment (hdhabu) [inline portfolio/src/main/webapp/script-charts.js:18]: Or rather '0'?

Comment (hdhabu) [reply portfolio/src/main/webapp/script-charts.js:18]: Oh wait, Row 0 might be the title Row :)

**Review by hdhabu — COMMENTED** [review]
Added a few comments inline with code

File: portfolio/src/main/java/com/google/sps/servlets/CharityDataServlet.java (hunk 1 above)
Comment (erichli1) [reply portfolio/src/main/java/com/google/sps/servlets/CharityDataServlet.java:87]: The limit check is to ensure that if it ever gets near the upper boundary, it won't wrap over to negatives

File: portfolio/src/main/webapp/script-charts.js (hunk 2 above)
Comment (erichli1) [reply portfolio/src/main/webapp/script-charts.js:19]: The first numbers are to indicate what row they should be on; I wanted both internships to be on the same row

**Review by hdhabu — APPROVED** [review]

**Pull request merged**